templates/install/install.tpl

```diff
@@ -136,9 +136,6 @@
 				{fbvFormSection label="installer.databaseDriver" description="installer.databaseDriverInstructions"}
 					{fbvElement type="select" id="databaseDriver" from=$databaseDriverOptions selected=$databaseDriver translate=false size=$fbvStyles.size.SMALL}
 				{/fbvFormSection}
-				{fbvFormSection label="installer.connectionCharset"}
-					{fbvElement type="select" id="connectionCharset" from=$connectionCharsetOptions selected=$connectionCharset translate=false size=$fbvStyles.size.SMALL}
-				{/fbvFormSection}
```

Comment:
Not quite true -- leaving it blank allows the DB to default!

Reply:
I took this decision partially based on this:
https://github.com/pkp/pkp-lib/blob/da5fe5542c79d0e8b5cdff2fd8a88c41a8896bab/classes/install/PKPInstall.inc.php#L108-L109

I'll double check later. By navigating in GitHub, looks like it will store "[Off](https://github.com/pkp/pkp-lib/blob/da5fe5542c79d0e8b5cdff2fd8a88c41a8896bab/classes/install/PKPInstall.inc.php#L218)" in the config, which will be restored as [false](https://github.com/pkp/pkp-lib/blob/52ff51c8e481dcbf6d83fc75103812dd25070b12/classes/config/ConfigParser.inc.php#L100), then I'm curious what happens when it's passed to [Laravel](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Database/Connectors/MySqlConnector.php#L69).

Reply:
Please do let me know what you turn up -- don't forget that we still have ADODB to consider, as long as we're still carrying the upgrade steps prior to 3.3.0; there will probably be considerations for ADODB as well as Laravel.

Reply:
Just to confirm, for now this setting value is really breaking things:
```php
PHP Fatal error: Uncaught PDOException: SQLSTATE[42000]:
Syntax error or access violation: 1115 Unknown character set: '' in /var/www/html/lib/pkp/lib/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:79
```